delfin/common/constants.py

```diff
@@ -62,3 +62,30 @@ class VolumeType(object):
     THIN = 'thin'
 
     ALL = (THICK, THIN)
+
+
+# Enumerations for alert severity
+class Severity(enum.Enum):
+    FATAL = 'Fatal'
+    CRITICAL = 'Critical'
+    MAJOR = 'Major'
+    MINOR = 'Minor'
+    WARNING = 'Warning'
+    INFORMATIONAL = 'Informational'
+    NOT_SPECIFIED = 'NotSpecified'
+
+
+# Enumerations for alert category
+class Category(enum.Enum):
+    FAULT = 'Fault'
+    EVENT = 'Event'
+    RECOVERY = 'Recovery'
+    NOT_SPECIFIED = 'NotSpecified'
+
+
+# Enumerations for resource type
+class ResourceType(enum.Enum):
```

Comment:
Why is resource type limited to only these values ( there can be n number of resource types ( power supply, fan, cpy, cache, memory, lun, pool etc..) . mapping this to these limted resource type will be hard. Do we need to map?

Reply:
Updated as per latest model